From 1067b1b18bc31a9718ce6cae4ab6734afb7faeb2 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?M=C3=A5rten=20Nordheim?= <marten.nordheim@qt.io>
Date: Mon, 25 Jul 2022 10:26:20 +0200
Subject: [PATCH 124/157] Use class scope to refer to static function

The call to metaObject() in the ctor may generate a warning since it
calls a virtual function during construction.

Amends be09628e151f26f602024bae6a957ffb27ac872d

Change-Id: I27472786b41624d582525c4260a54db61ee7ed16
Reviewed-by: Harald Sitter <sitter@kde.org>
Reviewed-by: Aleix Pol Gonzalez <aleixpol@kde.org>
(cherry picked from commit afbae9c2acf8d5c0846a87f03edfb9407672e9ee)
---
 src/platformsupport/linuxaccessibility/dbusconnection.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/platformsupport/linuxaccessibility/dbusconnection.cpp b/src/platformsupport/linuxaccessibility/dbusconnection.cpp
index 0e5af37e3f..cc734abc63 100644
--- a/src/platformsupport/linuxaccessibility/dbusconnection.cpp
+++ b/src/platformsupport/linuxaccessibility/dbusconnection.cpp
@@ -74,7 +74,7 @@ DBusConnection::DBusConnection(QObject *parent)
     if (!addressEnv.isEmpty()) {
         // Only connect on next loop run, connections to our enabled signal are
         // only established after the ctor returns.
-        metaObject()->invokeMethod(
+        QMetaObject::invokeMethod(
                 this,
                 [this, addressEnv] {
                     m_enabled = true;
-- 
2.38.1

